### PR TITLE
Add OCR to Scanned/SimpleResources

### PR DESCRIPTION
**Pull request by tpendragon**
Setting an OCR language on creation will make all future pages auto-run OCR as part of the derivative process.

Setting an OCR language after pages have been added will run OCR on every existing page and all future appended pages.

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -65,7 +65,7 @@ jobs:
     steps:
       - attach_workspace:
           at: '~/figgy'
-      - run: sudo apt-get install gdal-bin libgdal-dev libcairo2-dev libpango1.0-dev
+      - run: sudo apt-get install gdal-bin libgdal-dev libcairo2-dev libpango1.0-dev tesseract-ocr tesseract-ocr-ita tesseract-ocr-eng
```
Comment (hackmastera) [inline .circleci/config.yml]: Add ticket for corresponding ansible updates?

File: app/jobs/run_ocr_job.rb [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+  def perform(file_set_id)
+    file_set = query_service.find_by(id: Valkyrie::ID.new(file_set_id))
+    change_set = DynamicChangeSet.new(file_set)
+    derivative_service.new(change_set).create_derivatives
+  end
+
+  private
+
+    def derivative_service
```
Comment (hackmastera) [inline app/jobs/run_ocr_job.rb]: call this method derivative_service_factory? it's sort of confusing given the inconsistency with app/derivative-services/plum_derivative_service.rb#hocr_derivative_service

File: .circleci/config.yml (hunk 1 above)
Comment (hackmastera) [reply .circleci/config.yml]: Should merging this wait until ansible has been updated / run?

Comment (tpendragon) [reply .circleci/config.yml]: https://github.com/pulibrary/princeton_ansible/pull/249

**Pull request merged**